be/src/vec/jsonb/serialize.cpp

```diff
@@ -319,4 +319,22 @@ void JsonbSerializeUtil::jsonb_to_block(const TupleDescriptor& desc,
     }
 }
 
+// single row
+void JsonbSerializeUtil::jsonb_to_block(const TupleDescriptor& desc, const Slice& data,
```

Comment:
jsonb_to_block for single row can be reused by jsonb_to_block for multiple rows

Reply:
ok